idunn/blocks/environment.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import requests
+
+from apistar import validators
+from .base import BaseBlock
+from idunn import settings
+
+logger = logging.getLogger(__name__)
+
+
+
+class AirQuality(BaseBlock):
+    BLOCK_TYPE = "Environment"
+
+    air_quality = validators.Object(allow_null=True)
```

Comment:
here should be defined multiple fields such all air quality indicators (`o3`...), a global index (between 0 and 100), and the datetime of last measurement.